jubilant/_test_helpers.py

```diff
@@ -49,4 +53,14 @@ def juju():
     finally:
         if not keep:
             assert juju.model is not None
-            juju.destroy_model(juju.model, destroy_storage=True, force=True)
+            try:
```

Comment:
I think we should have a comment here that explains why we aren't using the provided method, so people don't need to blame to understand that.

I wonder if the doc for `temp_models should also explicitly say that it'll wait up to 10m to destroy.

Reply:
Both good points - added.